cpp/fetcher/fetcher.cc

```diff
@@ -146,7 +146,7 @@ void FetchState::WalkEntries() {
 
   int64_t index(start_);
   int num_fetch(0);
-  for (Range* current = entries_.get(); current;
+  for (Range *current = entries_.get(); current;
```

Comment:
It is :)
Weirdly, if we move `Range* current` out of the  `for` loop then it works as expected.
Bug in clang-format maybe?